src/hooks/veNft/VeNftResolverTokenUri.ts

```diff
@@ -0,0 +1,18 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber'
+
+import { useVeNftResolverContract } from 'hooks/veNft/VeNftResolverContract'
+
+import useV2ContractReader from '../v2/contractReader/V2ContractReader'
+
+export function useVeNftResolverTokenUri(
+  resolverAddress: string | undefined,
+  amount: BigNumber,
+  duration: BigNumberish,
+  lockDurationOptions: BigNumber[] | undefined,
+) {
+  return useV2ContractReader<string>({
+    contract: useVeNftResolverContract(resolverAddress),
+    functionName: 'tokenURI',
+    args: [1, amount, duration, 1, lockDurationOptions],
```

Comment:
**suggestion**: it's been helpful to add a lil comment next to each arg, with the name of the argument from the contract.

I'm not sure what `1` means in this context.

Example from launch project tx:

```js
    const args = [
      userAddress, // _owner
      [projectMetadataCID, JUICEBOX_MONEY_METADATA_DOMAIN], // _projectMetadata (JBProjectMetadata)
      fundingCycleData, // _data (JBFundingCycleData)
      fundingCycleMetadata, // _metadata (JBFundingCycleMetadata)
      mustStartAtOrAfter, // _mustStartAtOrAfter
      groupedSplits, // _groupedSplits,
      fundAccessConstraints, // _fundAccessConstraints,
      [contracts.JBETHPaymentTerminal.address], //  _terminals (contract address of the JBETHPaymentTerminal)
      DEFAULT_MEMO,
    ]
```

Reply:
Will do, but tbh i'm confused about this one too and I actually have a comment in my og dev branch wondering about these seemingly dummy args. I pulled the 1 value from the respective test. Will ask about them contract-side.

<img width="769" alt="image" src="https://user-images.githubusercontent.com/33093632/180779906-f29d18e1-c47a-4bdf-aa48-934ecf77c45a.png">

Reply:
Viraz says they are tokenID and lock end time respectively. I'm not sure if I'm able to fix that in this PR without subgraph access so I'm just gonna make an issue for it.

Reply:
Tracked in #1531